src/istep/edbgIstep.C

```diff
@@ -59,14 +59,14 @@ const edbgIPLTable::edbgIStep_t edbgIPLTable::cv_edbgIStepTable[] =
   { 0,   1,                                   "poweron",                   EDBG_ISTEP_NOOP},
```

Comment:
There is also a comment in the code mentioning version we need to update

Reply:
updated in help file , any additioan change?

Reply:
in the code 
line#27 //Updated this table as per P10_IPL_Flow_v0.91.04.

Reply:
Done